modules/augment/include/opencv2/augment/flip.hpp

```diff
@@ -35,6 +35,29 @@ namespace augment {
     };
 
 
+
+    class CV_EXPORTS_W FlipVertical : public Transform
+    {
+    public:
+        /* @brief Constructor
+           @param _probability probability that this transformation will be applied to each image
+        */
+        CV_WRAP FlipVertical(const Scalar& _probability);
```

Comment:
The transformation itself should not manage probability of applying it. It's more higher level.

So the constructor should accepts only parameters of the transformation. Nothing else.

Reply:
I think it this is the more convinient way for the python api, if we will implement an interface as easy as keras.

for example

```
augmenter = cv2.Augmenter()

augmenter.add( cv2.augment_FlipHorizontal(probability = 0.2) )

augmenter.add( cv2.augment_GaussianBlur(probability = 0.2) )

augmenter.apply(images)
```

don't you think?

Reply:
So the following one looks not more complicated:

```python
augmenter = cv2.Augmenter()
augmenter.add( cv2.augment_FlipHorizontal(), probability = 0.2 )
augmenter.add( cv2.augment_GaussianBlur(), probability = 0.2 )
augmenter.apply(images)
```

Reply:
Ok good, will try that.